### DISCUSSION TITLE
How to compute the Floating point operations (FLOPs) for a given model?

### DISCUSSION THREAD

**Discussion by KevinHooah** [pyg-team/pytorch_geometric#6812] [Q&A] [answered]
Hi, I want to ask once we build a model from PyTorch Geometric, how to measure the Floating point operations (FLOPs)?
I have tried the `summary` method which gives me the number of parameters in the model. Are there any handy packages/tools/libraries to measure FLOPs? Thank you!

**Comment by rusty1s** [comment, marked as answer]
Never did this before, so my advice may not be very useful. It looks like there exists some external tools that try to estimate FLOPs, e.g., https://github.com/1adrianb/pytorch-estimate-flops. Otherwise, PyTorch provides a set of benchmark utilities (but I am not sure if they support FLOP by now).
